pkg/operator/cassandra/controller/controller.go

```diff
@@ -139,9 +134,12 @@ func New(
 			}
 			cc.enqueueCluster(newCluster)
 		},
-		DeleteFunc: func(obj interface{}) {
-			// TODO: handle deletion
-		},
+		//Deletion handling:
+		// Atm, the only thing left behind will be the state, ie
+		// the PVCs that the StatefulSets don't erase.
```

Comment:
but if the PVCs had a reclaim policy of Delete, then they would have already been cleaned up by now?

Reply:
`reclaimPolicy` is a property of the `StorageClass` object and `persistentVolumeReclaimPolicy` is a property of the `PersistentVolume` object. What happens is, after a PVC is deleted, the reclaim policy for the specific volume gets invoked. In our case, the StatefulSets don't delete the PVCs, so the reclaim policy won't be invoked.